cmake/modules/FindGINKGO.cmake

```diff
@@ -63,6 +64,13 @@ FOREACH(_library ginkgo ${GINKGO_INTERFACE_LINK_LIBRARIES})
     )
 ENDFOREACH()
 
+#
+# Get ginkgo version number
+#
+IF(Ginkgo_FOUND)
```

Comment:
```suggestion
IF(GINKGO_FOUND)
```

Reply:
Sorry, but at this point the variable ``GINKGO_FOUND`` does not yet exists. If I see it correctly it is first defined in https://github.com/dealii/dealii/blob/b7a9c8420bfd8667c126878a80ce93833b3b15b8/cmake/macros/macro_deal_ii_package_handle.cmake#L60. Here I adopted the mechanism from https://github.com/dealii/dealii/blob/b7a9c8420bfd8667c126878a80ce93833b3b15b8/cmake/modules/FindBOOST.cmake.